src/retro_data_structures/formats/pak_wii.py

```diff
@@ -70,14 +70,70 @@ def _emitparse_header(code: construct.CodeGen) -> str:
     _resources_end=construct.Tell,
 )
 
-CompressedPakResource = FocusedSeq(
-    "data",
-    decompressed_size=Rebuild(Int32ub, len_(this.data)),
-    # Added Zlib check for DKCR
-    data=IfThenElse(game_check.uses_lzo, LZOCompressedBlock(this.decompressed_size), ZlibCompressedBlock),
+CMPD_Pak_Resource = Struct(
```

Comment:
I'd call this just `CMPD`

Maybe makes sense to move to it's own file? @duncathan

Reply:
just cmpd.py, don't think it's used for anything else so far?

Reply:
It isn't, no